app/react/kubernetes/cluster/NodeView/NodeApplicationsDatatable/columns.tsx

```diff
@@ -52,11 +52,19 @@ export function useColumns() {
         }),
         helper.accessor('CPU', {
           header: 'CPU reservation',
-          cell: ({ getValue }) => _.round(getValue(), 2),
+          cell: ({ row: { original: item } }) => (
+            <>
+              {_.round(item.Resource?.cpuRequest || 0, 2)}
+            </>
+          ),
         }),
```

Comment:
With the correct accessor used, getValue() should work again too, which is a cleaner way to grab the value where possible.